<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab</class>
 <widget class="QWidget" name="Tab">
  <property name="styleSheet">
   <string notr="true">QSplitter::handle {
   background: black;
}

QFrame#searchEditFrame, QFrame#tabBarFrame {
   border-top: none;
   border-right: none;
   border-bottom: 1px solid black;
   border-left: none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="searchEditFrame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="searchToolbar">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="SearchEdit" name="searchEdit">
            <property name="placeholderText">
             <string>Enter your query</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="indentation">
          <number>10</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="seeAlsoLabel">
         <property name="text">
          <string>See also</string>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="sections">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="viewInfo">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="tabBarFrame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="toolbox">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="backButton">
            <property name="text">
             <string>←</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="forwardButton">
            <property name="text">
             <string>→</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="toolbarLayout">
            <item>
             <widget class="QPushButton" name="openUrlButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Open URL</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="SearchableWebView" name="webView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/widgets/searchedit.h</header>
  </customwidget>
  <customwidget>
   <class>SearchableWebView</class>
   <extends>QWidget</extends>
   <header>ui/widgets/searchablewebview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
